frontend/packages/console-app/src/components/nodes/NodeDetailsOverview.tsx

```diff
@@ -57,8 +57,26 @@ const NodeDetailsOverview: React.FC<NodeDetailsOverviewProps> = ({ node }) => {
             <dd>
               <NodeIPList ips={getNodeAddresses(node)} expand />
             </dd>
-            <dt>{t('console-app~Node labels')}</dt>
-            <dd>
+            <dt>
+              <Split>
```

Comment:
I believe we can use the `DetailsItem` component to avoid copying this markup structure.

cc @TheRealJon

Reply:
Updated